cmake/toolchain.linux-x64-fuzzer.cmake

```diff
@@ -0,0 +1,32 @@
+# Toolchain for compiling with ASAN enabled on a Linux-x86-64 host.
```

Comment:
Where is this toolchain actually used? By CI? It isn't mentioned in the CMakePresets.json file.